security/cmd/node_agent/main.go

```diff
@@ -44,11 +44,20 @@ func init() {
 	flags.StringVar(&naConfig.IstioCAAddress,
 		"ca-address", "istio-ca:8060", "Istio CA address")
 	flags.StringVar(&naConfig.Env, "env", "onprem", "Node Environment : onprem | gcp | aws")
-	flags.StringVar(&naConfig.PlatformConfig.CertChainFile, "cert-chain",
+
+	flags.StringVar(&naConfig.PlatformConfig.OnPremConfig.CertChainFile, "onprem-cert-chain",
 		"/etc/certs/cert-chain.pem", "Node Agent identity cert file")
```

Comment:
Could you make the help message specific to the platform? Like: "Node Agent identity cert file in On Premise env", same for others.

Reply:
I will submit this PR once this is fixed :)

Reply:
Done. Thanks!!